test/jasmine/assets/custom_assertions.js

```diff
@@ -59,6 +59,92 @@ exports.assertHoverLabelStyle = function(g, expectation, msg, textSelector) {
     expect(textStyle.fill).toBe(expectation.fontColor, msg + ': font.color');
 };
 
+function assertLabelContent(label, expectation, msg) {
+    var lines = label.selectAll('tspan');
```

Comment:
in case we want to test styled labels at some point, `selectAll('tspan.line')`

Reply:
Good eyes. Thanks!

Reply:
done in 294714b